tests/mocks/rpc-handler.ts

```diff
@@ -22,17 +23,17 @@ export class RPCHandler implements HandlerInterface {
   constructor(config: HandlerConstructorConfig) {
     this._networkId = config.networkId;
     this._networkRpcs = networkRpcs[this._networkId];
-    this._networkName = networkNames[this._networkId];
+    this._networkName = networkIds[this._networkId];
     this._initialize(config);
   }
   public async getFastestRpcProvider(): Promise<JsonRpcProvider> {
-    if (this._networkId === 31337) {
-      this._provider = new JsonRpcProvider(LOCAL_HOST, this._networkId);
+    if (this._networkId === "31337") {
```

Comment:
Why do you change these to strings?

Reply:
The ids are strings because they are keys, I could coerce them into numbers but then it throws type errors in other types when trying to use them to index so it would mean having to get more creative with the types.

I did use numbers and got creative with the types but this felt like the cleaner approach

```ts
export const CHAINS_IDS = {
  "1": "ethereum-mainnet",
  "2": ...
  ```
Just my opinion/pref but I think them being strings provides better auto-complete as opposed to numbers but there's no real difference

Reply:
That makes sense. In case coercing back to number via `Number()` becomes tedious, consider initalizing an empty array and then assigning the chain names to the appropriate indicies in the array. Then you can use numbers (array index) as your key.

Reply:
It shouldn't be an issue we only need to cast it when instantiating the provider which right now is only done twice